index.js

```diff
@@ -646,11 +646,11 @@ const InstabugModule = {
    * @deprecated use {@link CrashReporting.reportJSException}
    * Send handled JS error object
    *
-   * @param errorObject   Error object to be sent to Instabug's servers
+   * @param Exception   Error object to be sent to Instabug's servers
    */
 
-  reportJSException(errorObject) {
-    CrashReporting.reportJSException(errorObject);
+  reportJSException(Exception) {
+    CrashReporting.reportJSException(Exception);
```

Comment:
Although Exception and Error are both synonymous, it's the Error term that dominates in JS. Can we update the TS file instead please?